packages/CodingStyle/src/Rector/ClassMethod/MakeInheritedMethodVisibilitySameAsParentRector.php

```diff
@@ -212,7 +214,7 @@ private function isStaticSelfFactory(ClassMethod $classMethod): bool
                 return false;
             }
 
-            return $this->isName($node->expr->class, 'self');
+            return $this->isName($node->expr->class, 'self') || $this->isName($node->expr->class, 'static');
```

Comment:
Better use early return for readability and consistence with the rest of the code

```php
if ($this->isName($node->expr->class, 'self')) {
    return true;
}

if ($this->isName($node->expr->class, 'static')) {
    return true;
}

return false;
```

Reply:
Will do next time I sit behind a laptop.